app/assets/scripts/config/staging.js

```diff
@@ -10,11 +10,11 @@ module.exports = {
       accessToken: 'pk.eyJ1IjoiaG90IiwiYSI6ImNpdmlkM2lkMDAwYTAydXBnNXFkd2EwemsifQ.KPrUb_mKlPmHCR6LNrSihQ'
     },
 
-    initialZoom: 8,
+    initialZoom: 3,
```

Comment:
If the staging settings are the same as production there's no need to repeat them.
`production.js` works as a base and the others are just overrides.